### PR TITLE
cc110x: make protocol configurable

### PR DESCRIPTION
**Pull request by OlegHahm**
Instead of hardcoding the protocol.

### CODE REVIEW

File: drivers/cc110x/cc110x-netdev2.c [hunk 1]
```diff
@@ -155,6 +151,15 @@ static int _set(netdev2_t *dev, netopt_t opt, void *value, size_t value_len)
                 return -EINVAL;
             }
             return 1;
+        case NETOPT_PROTO:
+            if (value_len != sizeof(gnrc_nettype_t)) {
+                return -EINVAL;
+            }
+            else {
+                memcpy(&(cc110x->proto), value, sizeof(gnrc_nettype_t));
```
Comment (kaspar030) [inline drivers/cc110x/cc110x-netdev2.c]: cast & assign instead of memcpy?

Comment (OlegHahm) [inline drivers/cc110x/cc110x-netdev2.c]: oops, copy & pasta shit - let's blame the jet lag

**Comment by OlegHahm** [comment]
fixed

**Comment by OlegHahm** [comment]
@kaspar030, ping!

**Comment by OlegHahm** [comment]
@kaspar030?

**Comment by OlegHahm** [comment]
@kaspar030?

**Comment by kaspar030** [comment]
Hey, sorry. Looks good but I don't have time to test. ACK for the change.

**Comment by OlegHahm** [comment]
Squashed.

**Comment by kaspar030** [comment]
Seems it also needs a rebase. But this is simple enough and not very central, so IMHO we can merge when travis is happy. @OlegHahm what do you think?

**Comment by OlegHahm** [comment]
And rebased.

**Comment by OlegHahm** [comment]
I agree.

**Comment by DipSwitch** [comment]
Travis is happy, so I'll just follow the orders ;)

**Pull request merged**